core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -86,7 +86,7 @@
 import static org.elasticsearch.cluster.metadata.IndexMetaData.SETTING_NUMBER_OF_REPLICAS;
 import static org.elasticsearch.cluster.metadata.IndexMetaData.SETTING_NUMBER_OF_SHARDS;
 import static org.elasticsearch.cluster.metadata.IndexMetaData.SETTING_VERSION_CREATED;
-import static org.elasticsearch.common.settings.Settings.settingsBuilder;
+import static org.elasticsearch.common.settings.Settings.builder;
```

Comment:
We probably should never `import static Strings.builder` because `Settings.builder()` is much more readable than `builder()`. It'd be fairly simple to add a check for that to the regexes we run during checkstyle. I'd probably do that - either temporarily to find and remove them all or permanently with a helpful message telling people to use `Settings.builder`.

BTW, I love static imports when the static methods are named appropriately, but I also love when static methods can be named in such a way that it is nicer **not** to statically import them. So, personally, I like that we don't have a hard and fast rule against static imports. The SETTINGS above this are a super great use of static imports.